lib/elastic_apm/central_config.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'elastic_apm/central_config/cache_control'
+
+module ElasticAPM
+  # @api private
+  class CentralConfig
+    include Logging
+
+    # @api private
+    class ResponseError < InternalError
+      def initialize(response)
+        @response = response
+      end
+
+      attr_reader :response
+    end
+    class NotFoundError < ResponseError; end
+    class ServerError < ResponseError; end
+
+    DEFAULT_MAX_AGE = 300
+
+    def initialize(config, default_polling_interval: 300)
```

Comment:
please reuse `DEFAULT_MAX_AGE` here

I am not sure `default_polling_interval` is really needed, as not used anywhere.